src/TwoStepBrownianFlow.h

```diff
@@ -192,19 +167,16 @@ void export_TwoStepBrownianFlow(pybind11::module& m, const std::string& name)
 
     py::class_<BrownianFlow, std::shared_ptr<BrownianFlow>>(m,
                                                             name.c_str(),
-                                                            py::base<TwoStepLangevinBase>())
+                                                            py::base<hoomd::md::TwoStepLangevinBase>())
         .def(py::init<std::shared_ptr<SystemDefinition>,
                       std::shared_ptr<ParticleGroup>,
                       std::shared_ptr<Variant>,
                       std::shared_ptr<FlowField>,
-                      unsigned int,
-                      bool,
-                      Scalar,
                       bool>())
-        .def("setFlowField", &BrownianFlow::setFlowField)
-        .def("setNoiseless", &BrownianFlow::setNoiseless);
+        .def("getFlowField", &BrownianFlow::getFlowField)
+        .def("getNoiseless", &BrownianFlow::getNoiseless);
```

Comment:
This is how you make a property. `flow_field` should be read-only, but `noiseless` can be mutable.
```suggestion
        .def_property_readonly("flow_field", &BrownianFlow::getFlowField)
        .def_property("noiseless", &BrownianFlow::getNoiseless, &BrownianFlow::setNoiseless);
```